packages/design-system/src/utils/constants.js

```diff
@@ -32,3 +32,62 @@ export const FOCUSABLE_SELECTORS = [
   'textarea:not(:disabled)',
   '[tabindex]:not([tabindex="-1"])',
 ];
+
```

Comment:
🤔 none of these feel like they are part of the design system. These are specific to the editor, no? 

The idea of a design system was that it's just shared components/theme.

Reply:
Agree, I included all of them in the list here to discuss https://github.com/GoogleForCreators/web-stories-wp/pull/10452#issuecomment-1059702590